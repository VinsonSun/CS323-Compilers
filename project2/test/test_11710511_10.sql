struct X {
	int v = 0;
}

int test1() {
	struct X x;
	int y = 1;
	y = y + x;
}
