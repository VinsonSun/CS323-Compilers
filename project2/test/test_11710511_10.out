Error type 10 at Line 3: indexing on non-array variable
